docs/docs-beta/docs/integrations/libraries/fivetran.md

```diff
@@ -18,19 +18,65 @@ tags: [dagster-supported, etl]
 ---
 
 
+# Using Dagster with Fivetran
```

Comment:
h1 header can be removed in `docs-beta` as it's derived from `title` front matter.

Reply:
Fixed in [d82cf47](https://github.com/dagster-io/dagster/pull/26485/commits/d82cf4702a85eb3a643a564963a333a4780aa091)